crates/stdarch-verify/tests/x86-intel.rs

```diff
@@ -140,6 +141,156 @@ fn verify_all_signatures() {
 
     let mut all_valid = true;
     'outer: for rust in FUNCTIONS {
+        if !rust.has_test {
+            // FIXME: this list should be almost empty
+            let skip = [
+                "__readeflags",
```

Comment:
We should try to make this list smaller over time. It might well be that some of these are already tested in other tests, or that the test names have typos or similar.